bin/setup

```diff
@@ -34,147 +35,148 @@ CONTAINER_TAG=`cat ${ROOT}/bin/.container-tag`
 #  3. Docker, if DOCKER is set in the environment
 #  4. Singularity, if SINGULARITY is set in the environment
 #  5. The choice made by the user, which will be cached in .container-type
-if [ -v DEVCONTAINER_ENV ]; then
-CONTAINER_TYPE=devcontainer
-elif [ -f ${ROOT}/.container-type ]; then
-CONTAINER_TYPE=`cat ${ROOT}/.container-type`
-elif [ -v DOCKER ]; then
-CONTAINER_TYPE=docker
-elif [ -v SINGULARITY ]; then
-CONTAINER_TYPE=singularity
+if [ -n "${DEVCONTAINER_ENV+x}" ]; then
+  CONTAINER_TYPE=devcontainer
+elif [ -f "${ROOT}/.container-type" ]; then
```

Comment:
Why do we need `{}` around `ROOT`? I guess you're just carrying that from the unchanged code?

Reply:
@ThinkOpenly this is on picking one style (with or without braces) and use it consistently.  The ${} around ROOT here are just a style choice and aren’t strictly needed. ${ROOT} and $ROOT behave the same unless we are appending characters right after the variable name.